README.md

```diff
@@ -46,7 +46,7 @@ person.sayName().changeName("Timmy").sayName();
 
 > 2016-01-19 by [@gogainda](https://twitter.com/gogainda)
 
-Suppose you have a couple of variables with unknown types and you want to concatenate them. To be sure that the arithmetical operation would not be applied during the concatenation:
+Suppose you have a couple of variables with unknown types and you want to concatenate them in a string. To be sure that the arithmetical operation is not be applied during concatenation, use ```concat```:
```

Comment:
I just copied that from further down (lines 68 and 70). I wasn't trying to fix code, just grammar. But I can clean the back-ticks up in a new commit.